deploy.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+bundle install
+bundle exec middleman build
+aws s3 cp --recursive build/ s3://com.getdbt.hub/
```

Comment:
Are files ever deleted/omitted via the build process? If so, I think just `cp`ing will leave the files that weren't rebuilt in the remote bucket

Reply:
If so, `aws s3 sync --delete` might be a quick option, or always building into a new s3 path (prefix with some deploy identifier) and then switch the cloudfront distribution to read from the new prefix 

Neither need to hold us up today.

Reply:
there's enough of a chance that the build fails and then sync deletes files that I don't want to use that. the semantics of this are such that files are rarely or never deleted. I'd rather clean that up down the road (and put a proper staging env in place to test)

Reply:
I will add pipefail directives though so that this script exits early if the early build step fails

Reply:
Roger